docs/en/reference/information_schema/materialized_views.md

```diff
@@ -0,0 +1,34 @@
+---
+displayed_sidebar: "English"
+---
+
+# materialized_views
+
+`materialized_views` provides information about all asynchronous materialized views.
+
+The following fields are provided in `materialized_views`:
+
+| **Field**                            | **Description**                                              |
+| ------------------------------------ | ------------------------------------------------------------ |
+| MATERIALIZED_VIEW_ID                 | ID of the materialized view.                                 |
+| TABLE_SCHEMA                         | Database in which the materialized view resides.             |
+| TABLE_NAME                           | Name of the materialized view.                               |
+| REFRESH_TYPE                         | Refresh type of the materialized view, including `ROLLUP`, `ASYNC`, and `MANUAL`. |
+| IS_ACTIVE                            | Indicates whether the materialized view is active. Inactive materialized views can not be refreshed or queried. |
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/8cd62054-e3fb-41bd-b6fe-60611948e3a6)
suggest changing to "cannot"